src/__init__.py

```diff
@@ -221,11 +221,19 @@ def run_migrate(clients: Clients, user_inputs: UserInputs) -> MigrateOutputs | N
 
     # Check difference with main
     changes = recreate_docs(clients, DOCUMENTATION_TAG)
+    # Check whether there are still changes after switching to the base branch
+    if changes:
+        changes = clients.repository.is_dirty(user_inputs.base_branch)
+
+        # Move the tag if there are no changes
+        if not changes:
+            with clients.repository.with_branch(user_inputs.base_branch) as repo:
+                main_hash = repo.current_commit
+            clients.repository.tag_commit(DOCUMENTATION_TAG, main_hash)
+
     if not changes:
         logging.info(
-            "No community contribution found in commit %s. Discourse is inline with %s",
-            user_inputs.commit_sha,
-            DOCUMENTATION_TAG,
+            "No community contribution found, discourse is inline with %s", DOCUMENTATION_TAG
```

Comment:
can this be merged under `if not changes:` line above?

Reply:
The `if not` above is in another `if` block, or do you mean another location?